app/views/steps/capital/commercial_property/edit.html.erb

```diff
@@ -24,7 +24,7 @@
       <% end %>
       <%= f.govuk_collection_radio_buttons :has_other_owners, YesNoAnswer.values, :value, inline: true, legend: { size: 's' } %>
 
-      <%= f.continue_button %>
+      <%= f.continue_button(secondary: false)  %>
```

Comment:
I will also add a comment explaining why we have made the change to the form if we agree to go ahead with removing the button